lib/src/default_index_store.rs

```diff
@@ -2701,6 +2712,62 @@ mod tests {
         );
     }
 
+    #[test]
+    #[allow(clippy::redundant_clone)] // allow id_n.clone()
```

Comment:
Yeah, it can be moved to `mod tests {}` scope (if "all tests" means that.)
These tests follow similar patterns, so it would make sense to suppress warning at module level.